utils/land-patterns.stanza

```diff
@@ -9,23 +9,120 @@ defpackage ocdb/land-patterns :
     prefix(pad) => def-
   import ocdb/design-vars
 
+;==============================================================================
+;============================== Helpers =======================================
+;==============================================================================
 ; ====== Argument constraint Helpers ===========================================
+; Ensure that a value is even and positive.
 defn ensure-even-positive! (value: Int, name: String):
   if value < 2 or value % 2 == 1:
     fatal("%_ only supports a positive even number of pins." % [name])
 
 ; ====== Reference Label Helpers ===============================================
-public defn ref-label (x:Double, y:Double, a:Anchor) :
-  inside pcb-landpattern :
-    layer(Silkscreen("values", Top)) = Text(">REF", 0.7056, a, loc(x,y))
-    
+; Creates a reference label from pose and anchor
+public defn ref-label (pose:Pose, anchor:Anchor):
+  inside pcb-landpattern:
+    layer(Silkscreen("values", Top)) = Text(">REF", MIN-SILKSCREEN-TEXT-HEIGHT, anchor, pose)
+
+; Creates a reference label at a fixed pose
+public defn ref-label (pose:Pose):
+  ref-label(pose, C)
+
+; DEPRECATED: prefer ref-label(loc(x, y))
+; Creates a reference label at an x,y position and anchor
+public defn ref-label (x:Double, y:Double, anchor:Anchor):
+  ref-label(loc(x, y), anchor)
```

Comment:
should we print a warning about the deprecation?

Reply:
we have no good way of doing that at the moment. We can print a warning message, but users will either ignore it or treat it as an error (or it will be lost in the rest of our output).